Sources/Subs-Db-postgresql.php

```diff
@@ -966,18 +968,18 @@ function smf_db_fetch_all($request)
  */
 function smf_db_error_insert($error_array)
 {
-	global  $db_prefix, $db_connection;
+	global $db_prefix, $db_connection;
 	static $pg_error_data_prep;
 
 	// without database we can't do anything
 	if (empty($db_connection))
 		return;
 
 	if (empty($pg_error_data_prep))
-			$pg_error_data_prep = pg_prepare($db_connection, 'smf_log_errors',
-				'INSERT INTO ' . $db_prefix . 'log_errors(id_member, log_time, ip, url, message, session, error_type, file, line, backtrace)
-													VALUES(		$1,		$2,		$3, $4, 	$5,		$6,			$7,		$8,	$9, $10)'
-			);
+		$pg_error_data_prep = pg_prepare($db_connection, 'smf_log_errors',
+			'INSERT INTO ' . $db_prefix . 'log_errors(id_member, log_time, ip, url, message, session, error_type, file, line, backtrace)
```

Comment:
Again, a line break and an extra tab between `log_errors` and the parenthesis would make this much more legible.